src/commands/module/_utils.ts

```diff
@@ -71,3 +71,12 @@ export async function fetchModules(): Promise<NuxtModule[]> {
   )
   return data
 }
+
+export async function validateNpmModule(module: string): Promise<boolean> {
+  try {
+    await $fetch(`https://registry.npmjs.org/${module}`)
```

Comment:
One thing i am worried is that this way we don't support private packages/registries. We might only use a soft warning to say hey we couldn't validate the package from public npm registry, it might go wrong.